### DISCUSSION TITLE
How can I use a hext .txt file with 7th Heaven?

### DISCUSSION THREAD

**Discussion by GrumpyGourmand** [tsunamods-codes/7th-Heaven#148] [Q&A] [answered]
Hello,

I'm trying to use a simple hext patch for Final Fantasy VII (save & PHS anywhere).  With respect to FFNx, I know that I would simply drop the hext .txt file into the .\hext\ff7\en\ folder and it would be loaded automatically.  The problem is, 7th Heaven deletes the .\hext folder upon clicking 'Play'.  That tells me that 7th Heaven does not want me using hext files in this way (potentially because they could interfere with mods?).  Deleting the folder seems like an extreme solution, as it gets deleted regardless of what it contains (so if I had a personal collection of hext files, or anything else, it would all be destroyed) and with no warning or notification of any kind.

All of that being said, how am I meant to load a hext file for use with 7th Heaven?  Do I really have to create my own mod to incorporate just a hext file?  I tried searching for this issue to no avail.  If anyone could enlighten me regarding this, I would be very appreciative.

Edit: I have since found that there is a mod called Gameplay Tweaks which contains the functionality I was looking for (save & PHS anywhere), but my question about how to use hext files with 7th Heaven and my concern about deleting the hext folder still remains.

**Comment by julianxhokaxhiu** [comment, marked as answer]
There is already a Mod on 7th that allows you to enable that feature. You don't need to do anything in particular. But please hop on Discord for proper support, we do not provide 7th support over here :) Thanks
